geomesa-spark/geomesa-spark-jts/src/main/scala/org/locationtech/geomesa/spark/jts/udf/GeometricProcessingFunctions.scala

```diff
@@ -61,16 +61,23 @@ object GeometricProcessingFunctions {
     fastCircleToGeom(new JtsPoint(p, spatialContext).getBuffered(degrees, spatialContext))
   }
 
+  val ST_MakeValid: Geometry => Geometry = nullableUDF(geom => {
+    val fix = new util.GeometryFixer(geom)
+    fix.getResult
```

Comment:
GeometryFixer allows to set two arguments
- isKeepCollapsed (defaults to `false`)
- isKeepMulti (defaults to `true`)

I decided to go with the default values to keep it simple.